src/Controller/UserActivateController.php

```diff
@@ -25,7 +25,7 @@ public function __construct(
 
     public function __invoke(Request $request): JsonResponse
     {
-        $body = json_decode($request->getContent());
+        $body = json_decode($request->getContent(), null, 512, JSON_THROW_ON_ERROR);
```

Comment:
`Request::toArray` is your friend (cf. https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/HttpFoundation/Request.php#L1549-L1566).